src/lib/php-get-theme-details.ts

```diff
@@ -1,37 +1,42 @@
-import * as Sentry from '@sentry/electron/main';
+import { z } from 'zod';
 import SiteServerProcess from 'src/lib/site-server-process';
 
+const themeDetailsSchema = z.object( {
+	name: z.string().catch( '' ),
+	path: z.string(),
+	slug: z.string(),
+	isBlockTheme: z.boolean(),
+	supportsWidgets: z.boolean(),
+	supportsMenus: z.boolean(),
+} );
+
 export async function phpGetThemeDetails(
 	server: SiteServerProcess
 ): Promise< StartedSiteDetails[ 'themeDetails' ] > {
 	if ( ! server.php ) {
 		throw Error( 'PHP is not instantiated' );
 	}
 
-	let themeDetails = null;
 	const themeDetailsPhp = `<?php
 	require_once('${ server.php.documentRoot }/wp-load.php');
 	$theme = wp_get_theme();
 	echo json_encode([
 		'name' => $theme->get('Name'),
-		'uri' => $theme->get('ThemeURI'),
```

Comment:
In case it wasn't already clear, we don't actually use this data. `uri` is not included in the `StartedSiteDetails[ 'themeDetails' ]` type definition.